### PR TITLE
On errors print the playbook name

### PR DESCRIPTION
**Pull request by agx**
not the hardcoded site.yml

### CODE REVIEW

File: lib/ansible_spec/load_ansible.rb [hunk 1]
```diff
@@ -204,7 +204,7 @@ def self.load_dependencies(role)
 
       if File.exist?(path)
         new_deps = YAML.load_file(path).fetch("dependencies", []).map { |h|
-          h["role"]
+          h["role"] || h
```
Comment (volanja) [inline lib/ansible_spec/load_ansible.rb]: this change is requested by #83 
Please delete in this PR.

File: lib/ansible_spec/load_ansible.rb [hunk 2]
```diff
@@ -221,7 +221,7 @@ def self.load_playbook(f)
 
     # e.g. comment-out
     if playbook === false
-      puts 'Error: No data in site.yml'
+      puts 'Error: No data in #{f}'
```
Comment (volanja) [inline lib/ansible_spec/load_ansible.rb]: this change is looks good.

**Comment by agx** [comment]
On Thu, Sep 29, 2016 at 08:43:07AM -0700, volanja wrote:
> volanja requested changes on this pull request.
> 
> Delete another PR code.
> 
> > @@ -204,7 +204,7 @@ def self.load_dependencies(role)
>  
>        if File.exist?(path)
>          new_deps = YAML.load_file(path).fetch("dependencies", []).map { |h|
> -          h["role"]
> +          h["role"] || h
> 
> this change is requested by #83 
> Please delete in this PR.

Aargh, git commit -a vs git commit -p. Fixed.

> 
> > @@ -221,7 +221,7 @@ def self.load_playbook(f)
>  
>      # e.g. comment-out
>      if playbook === false
> -      puts 'Error: No data in site.yml'
> +      puts 'Error: No data in #{f}'
> 
> this change is looks good.

**Comment by volanja** [comment]
Thank you!
I merged it.

**Comment by volanja** [comment]
I mistaked check code.
So I modify correct code in #85

**Pull request merged**